### ISSUE TITLE
docker farmer - ends without any information

### ISSUE THREAD

**Issue by lippoliv** [Storj/storjshare-daemon#201]
### Package Versions

Replace the values below using the output from `storjshare --version`.

```
daemon: 3.4.2, core: 6.6.0, protocol: 1.1.0
```

Replace the values below using the output from `node --version`.

```
v6.11.0
```

### Expected Behavior

Farmer should start.

### Actual Behavior

Farmer starts and Ends, without any info.

```
storj_1  |   * starting daemon in background
storj_1  |
storj_1  |   * starting share with config at /storj/config.json
storj_storj_1 exited with code 0
```

Also Setting `"loggerVerbosity": 4` does not provide any additional info.

### Steps to Reproduce

Please include the steps the reproduce the issue, numbered below. Include as
much detail as possible.

1. Setup new docker runner following here: https://store.docker.com/community/images/computeronix/storjshare-daemon
2.Choose ubuntu
3.try to run as described

**Comment by lippoliv** [comment]
@aleitner can you please have a look? I am willing to add > 2TB to storj, but it has to run with docker. (sorry for mentioning directly, I know it's not the right way, but I need some feedback over here)

**Comment by lippoliv** [comment]
OK I found out that it'll run with `storjshare daemon start && storjshare start -c /storj/config.json && sleep 604800` so it'll run for one week and I Setup an `docker-compose restart` cron-job for 0AM on sunday. But that is not that nice.

**Comment by littleskunk** [comment]
try it with `storjshare daemon --foreground` and run the storjshare start command in a second terminal.

**Comment by lippoliv** [comment]
As I run in Docker there should just be one process. Can I run the daemon in a second Container? But then the worker still will fail I think. Can the `storjshare start` be connected to an "remote" daemon?

Giving a good Docker-Example should be a good idea for the Project. Everyone can test storj-nodes without fear. If they dislike, they can just remove the Container without trashing the System.

(If I have it up and running nice, I could send an Pull Request with an docker-compose.yml and an readme thatfor if you like)

**Comment by littleskunk** [comment]
` storjshare daemon --foreground &` should work as well and will allow you to run storjshare start in the same terminal.

**Comment by lippoliv** [comment]
nice idea, but sadly that doesn work with the `docker-compose.yml`

```
storj_1  | sh: 1: Syntax error: "&&" unexpected
```

**Comment by littleskunk** [comment]
Please check the daemon logfile `~/.config/storjshare/logs/daemon.log`

**Comment by calvintam236** [comment]
I made my own version docker image for `storjshare-daemon` as well. ([Link](https://hub.docker.com/r/calvintam236/storjshare/), build on latest version v3.5.5)

I had tried to run the daemon and nodes in separate containers, and connect them using the same docker network (not default `bridged`), it won't work as the node says it can't find the daemon.

I gave up and made the nodes to run detached mode.
However, the nodes will stop working after awhile (no console output at all for 5mins+ = probably dead?), so I need to restart the containers manually...
